posts/covid-analysis/index.qmd

```diff
@@ -0,0 +1,184 @@
+---
+title: "How well prepared are we to rapidly analyse a new influenza pandemic? A brief perspective on analysis conducted for UK government advisory groups during COVID-19"
+author:
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-11-28"
```

Comment:
```suggestion
date: "2024-12-23"
```

Or any other date you would prefer, as right before holiday season can make things fall into a void 😊 Don't want your writing to get lost!

Reply:
Thanks! That date is fine – suspect we get relatively little direct traffic so can share on networks once everyone has recovered from festivities...